server/src/test/java/e2e/StandaloneTestBootstrapper.java

```diff
@@ -39,10 +37,6 @@ public void setup() throws Exception {
         });
         kafka.start();
         workerConfig = new LHConfig(testClientProperties());
-        client = workerConfig
```

Comment:
@eduwercamacaro maybe this mock is needed 😅

Reply:
I believe it is not needed anymore right? LHConfig already sets the TENANT_ID. @eduwercamacaro